README.md

```diff
@@ -51,10 +51,17 @@ Transformers (keras/sklearn style): [Repo2nBOWTransformer](ast2vec/repo2/nbow.py
 
 ## Docker image
 
+First you need to run [babelfish](https://github.com/bblfsh/bblfshd) image and download drivers:
+```
+docker run -d --privileged -p 9432:9432 --name bblfshd bblfsh/bblfshd:v2.2.0 --log-level DEBUG
```

Comment:
As was agreed on one of SoS meetings, we should put the link to the official bblfsh installation manual.

Also, please remove `--log-level DEBUG`.